Automattic/sensei
# Add quiz submission tables based repository

Part of #5434

### Changes proposed in this Pull Request

* Add custom tables based repositories for quiz submission.
* Add aggregate repositories that use both types of repositories (comments based and tables based).

### Testing instructions

* Make a course a lesson and a quiz.
* Submit the quiz.
* Make sure there is an entry in the `wp_sensei_lms_quiz_submissions` table.
* Grade the quiz.
* Make sure the `final_grade` column is updated for the `wp_sensei_lms_quiz_submissions` entry.
* When resetting the quiz or resetting the course, the table entry should be deleted.
